pkg/api/handlers/toolreferences.go

```diff
@@ -108,6 +112,8 @@ func (a *ToolReferenceHandler) Create(req api.Context) (err error) {
 		return apierrors.NewBadRequest("reference is required")
 	}
 
+	newToolReference.Reference = strings.TrimPrefix(strings.TrimPrefix(newToolReference.Reference, "https://"), "http://")
```

Comment:
@StrongMonkey Sorry, I just noticed that we probably don't want to do this. It is valid for tool references to start with `http(s)://`. For example, non-GitHub-based tools.

Reply:
Should be fixed.